jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
Currently it only uses that folder since all "unsigned" .app files are written to "debug" folder.

We could get this information by running `xcodebuild -showBuildSettings`, but I think we can implement that command in the plugin so it can figure the build path out on its own.